extensions/cache/src/main/resources/benchmark-stream-operations.xml

```diff
@@ -0,0 +1,132 @@
+   <configurations>
+
+      <config name="ISPN9 - hotrod">
+         <setup group="server" plugin="infinispan90">
+            <server xmlns="urn:radargun:plugins:infinispan90:3.0" file="streaming.xml"
+               jmx-domain="jboss.datagrid-infinispan" start-timeout="120000" cache-manager-name="clustered">
+               <args>
+                  -Djava.net.preferIPv4Stack=true
+                  -Djboss.node.name=node${slave.index}
+                  -Djboss.socket.binding.port-offset=${slave.index}00
+               </args>
+               <home>${RADARGUN_WORK}/slave${slave.index}</home>
+               <server-zip>${ISPN_ZIP_PATH}/infinispan-server-9.0.0-SNAPSHOT.zip</server-zip>
+               <env>JAVA_OPTS=-server -Xms2g -Xmx4g -XX:NewRatio=3
+                  -XX:+UseConcMarkSweepGC
+                  -XX:+UseParNewGC
+                  -XX:MaxPermSize=4G
+                  -XX:+UseLargePages -verbose:gc
+                  -XX:+PrintGCDetails -XX:+PrintGCTimeStamps
+                  -XX:HeapDumpPath=/tmp/
+                  -XX:+HeapDumpOnOutOfMemoryError
+               </env>
+            </server>
+         </setup>
+         <setup group="client" plugin="infinispan90">
+            <hotrod xmlns="urn:radargun:plugins:infinispan90:3.0">
+               <servers>127.0.0.1:11222</servers>
+            </hotrod>
+            <vm-args>
+               <memory max="1G"></memory>
+            </vm-args>
+         </setup>
+      </config>
+
+
+   </configurations>
+
+   <!-- Sequence of stages executed on the cluster -->
+   <rg:scenario xmlns:rg="urn:radargun:benchmark:3.0" xmlns="urn:radargun:stages:core:3.0"
+      xmlns:cache="urn:radargun:stages:cache:3.0" xmlns:l="urn:radargun:stages:legacy:3.0">
+      <!-- Start services on all nodes -->
+      <service-start />
+      <!-- Begin monitoring of CPU, memory usage and GC -->
+      <jvm-monitor-start />
+
+      <!--Preload the cache with data -->
+      <cache:load num-entries="100" groups="client" load-all-keys="true">
+         <cache:cache-selector>
+            <cache:use-cache cache="streamed"></cache:use-cache>
+         </cache:cache-selector>
+      </cache:load>
+
+      <l:streaming-operations-test test-name="warmup"
+         num-threads-per-node="10" duration="60s" groups="client" entry-size="100000">
+         <l:value-generator>
+            <l:randomStream />
+         </l:value-generator>
+         <l:cache-selector>
+            <l:use-cache cache="streamed" />
+         </l:cache-selector>
+         <l:key-selector>
+            <l:concurrent-keys>
+               <l:total-entries>100</l:total-entries>
+            </l:concurrent-keys>
+         </l:key-selector>
+         <statistics>
+            <basic>
+               <operation-stats>
+                  <basic />
+               </operation-stats>
+            </basic>
+         </statistics>
+      </l:streaming-operations-test>
+
+      <rg:repeat name="StreamOperationsTest" from="1" to="11" inc="1">
```

Comment:
The loop is done on increasing value sizes (counter*100kb to remain reasonable on single machine), but cutting it by half or two thirds would not decrease its exemplary value, good enough?